corehq/warehouse/migrations/0023_auto_20180306_1258.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are two migrations that are prefixed with ```0023``` - can you please fix?  Is blocking staging deploy.

Reply:
bump @calellowitz @sravfeyn

Reply:
https://github.com/dimagi/commcare-hq/pull/19725